tests/components/device_tracker/test_init.py

```diff
@@ -117,11 +117,21 @@ async def test_reading_yaml_config(
     await hass.async_add_executor_job(
         legacy.update_config, yaml_devices, dev_id, device
     )
-    assert await async_setup_component(hass, device_tracker.DOMAIN, TEST_PLATFORM)
-    config = (await legacy.async_load_config(yaml_devices, hass, device.consider_home))[
-        0
-    ]
-    await hass.async_block_till_done()
+    loaded_config = None
+    original_async_load_config = legacy.async_load_config
+
+    async def capture_load_config(*args, **kwargs):
```

Comment:
What's the point of this patch?

Reply:
We loaded the config in the code, and we loaded the config in the test.  We don't do it twice now and we verify what the code actually loads, not what the test loads

Reply:
Ok.